docs/registries.conf.5.md

```diff
@@ -0,0 +1,25 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is TOML.
+
+# FORMAT
+The [TOML_format][toml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`.
+
+The following example configuration defines two registries and one insecure registries.
+
+```
+[registries]
+secure = ["registry1.com", "registry2.com"]
+insecure = ["registry3.com"]
+```
+
```

Comment:
You've 'blocked' below too.  Should that be talked about?  If so, it might be a good example for showing an empty set.